src/pydata_sphinx_theme/assets/styles/_color.scss

```diff
@@ -0,0 +1,274 @@
+  --pst-color-admonition-tip: var(--pst-color-success);
+  --pst-color-admonition-important: var(--pst-color-success);
+  --pst-color-admonition-background: var(--pst-color-background);
+  --pst-color-admonition-shadow: var(--pst-color-shadow);
+
+  // viersion modified
+  --pst-color-versionmodified-default: var(--pst-color-info);
+  --pst-color-versionmodified-added: var(--pst-color-success);
+  --pst-color-versionmodified-changed: var(--pst-color-warning);
+  --pst-color-versionmodified-deprecated: var(--pst-color-danger);
+
+  // preformatted
+  --pst-color-preformatted-text: 34, 34, 34;
+  --pst-color-preformatted-border: var(--pst-color-border);
+  --pst-color-preformatted-background: var(--pst-color-background-up);
+  --pst-color-preformatted-shadow: var(--pst-color-shadow);
+
+  //////////////////////////////////////////////////////////////////////
+  // layout
+  //////////////////////////////////////////////////////////////////////
+
+  // navbar
+  --pst-color-navbar-background: var(--pst-color-background);
+  --pst-color-navbar-link: var(--pst-color-deactivate);
+  --pst-color-navbar-link-hover: var(--pst-color-active-navigation);
+  --pst-color-navbar-link-active: var(--pst-color-active-navigation);
+
+  // toc
+  --pst-color-toc-link: var(--pst-color-deactivate);
+  --pst-color-toc-link-hover: var(--pst-color-active-navigation);
+  --pst-color-toc-link-active: var(--pst-color-active-navigation);
+  --pst-color-toc-border: var(--pst-color-border);
+
+  // sidebar
+  --pst-color-sidebar-link: var(--pst-color-deactivate);
+  --pst-color-sidebar-link-hover: var(--pst-color-active-navigation);
+  --pst-color-sidebar-link-active: var(--pst-color-active-navigation);
+  --pst-color-sidebar-expander-background-hover: var(--pst-color-background-up);
+  --pst-color-sidebar-caption: var(--pst-color-text-base);
+  --pst-color-sidebar-border: var(--pst-color-border);
+
+  // footer
+  --pst-color-footer-border: var(--pst-color-border);
+
+  // alert
+  --pst-color-alert-link: 232, 62, 140; // higher contrast of links in info boxes
+
+  // prev-next
+  --pst-color-prev-next: var(--pst-color-primary);
+  --pst-color-prev-next-btn: var(--pst-color-deactivate);
+
+  // search
+  --pst-color-search-background: var(--pst-color-background);
+  --pst-color-search: var(--pst-color-border);
+  --pst-color-search-border: var(--pst-color-border);
+
+  // ethical add
+  --pst-color-add-text: var(--pst-color-text-base);
+  --pst-color-add-background: var(--pst-color-background);
+  --pst-color-add-border: var(--pst-color-border);
+}
+
+/////////////////////////////////////////////////////////////////////////
+// dark theme
+/////////////////////////////////////////////////////////////////////////
+body[data-theme="dark"] {
+  //////////////////////////////////////////////////////////////////////
+  // theme colors
+  //////////////////////////////////////////////////////////////////////
+
+  // main colors
+  --pst-color-primary: 213, 142, 72;
```

Comment:
also just a general note that I don't feel it's necessary to prefix every single color variable with `--pst`. I suppose we are worried about name clashes, but it would be pretty unlikely, and maybe confusing, if there were two different variables like both `--pst-color-success` and `--color-success`

I guess that you are just moving around colors though, not adding new ones, right? If so that is probably out-of-scope for this PR

Reply:
yes, I used the same naming convention as the one already here. I also decided that it was safer to reuse existing names in case people were already overriding them in their documentation. I think it could be an idea but maybe for another PR. 
Can I mark this conversation as resolved?

Reply:
yep you're correct, we should just resolve this in a different PR